### PR TITLE
Harmonize Terraform across different CSP

### PR DESCRIPTION
**Pull request by mpagot**
Ticket: https://jira.suse.com/browse/TEAM-7678

# Verification

###  - sle-15-SP5-Qesap-Azure-Payg-x86_64-BuildLATEST_AZURE_SLE15_5_PAYG-qesap_azure_saptune_test
- http://openqaworker15.qa.suse.cz/tests/296120 :green_apple: 

### sle-15-SP6-Qesap-Aws-Byos-x86_64-BuildLATEST_AWS_SLE15_6_BYOS-qesap_aws_r48xlarge_test
- http://openqaworker15.qa.suse.cz/tests/296119 :green_apple: 
  
###  sle-15-SP5-Qesap-Gcp-Byos-x86_64-BuildLATEST_GCE_SLE15_5_BYOS-qesap_gcp_sbd_test
- http://openqaworker15.qa.suse.cz/tests/296121 :green_apple:  terraform part is fine
- http://openqaworker15.qa.suse.cz/tests/296123

### CODE REVIEW

File: terraform/aws/infrastructure.tf [hunk 1]
```diff
@@ -109,8 +109,8 @@ resource "aws_security_group" "secgroup" {
   vpc_id = local.vpc_id
 
   tags = {
-    Name      = "${local.deployment_name}-sg"
-    Workspace = local.deployment_name
+    name      = "${local.deployment_name}-sg"
+    workspace = local.deployment_name
```
Comment (BillAnastasiadis) [inline terraform/aws/infrastructure.tf:113]: In the `terraform/aws/modules` dir, all module (hana_node etc) `main.tf` files also have those defined, like

```
Name      = "${var.common_variables["deployment_name"]}-drbd-subnet-${count.index + 1}"
    Workspace = var.common_variables["deployment_name"]
    ```
    are they fine with the capital letters there?

Comment (mpagot) [reply terraform/aws/infrastructure.tf:113]: > In the `terraform/aws/modules` dir, all module (hana_node etc) `main.tf` files also have those defined, like
> 
> ```
> Name      = "${var.common_variables["deployment_name"]}-drbd-subnet-${count.index + 1}"
>     Workspace = var.common_variables["deployment_name"]
> ```
> 
> are they fine with the capital letters there?

no, let em fix it

File: terraform/aws/variables.tf [hunk 2]
```diff
@@ -776,20 +776,13 @@ variable "netweaver_shared_storage_type" {
...
   default     = false
 }
 
-# Pre deployment
-
-variable "pre_deployment" {
-  description = "Enable pre deployment local execution. Only available for clients running Linux"
-  type        = bool
-  default     = false
-}
```
[the remark this answers is not in the document]
Comment (mpagot) [reply terraform/aws/variables.tf:792]: no, was a feature needed by SALT to generate ssh key pair. I also totally deleted two terraform files related to that in this PR

**Review by BillAnastasiadis — APPROVED** [review]
LGTM

**Pull request merged**